src/test/java/org/opensearch/integTest/ToolIntegrationTest.java

```diff
@@ -47,7 +47,7 @@ public void setupTestAgent() throws IOException, InterruptedException {
             connectorId = setUpConnector();
         } catch (Exception e) {
             // Wait for ML encryption master key has been initialized
-            TimeUnit.SECONDS.sleep(10);
+            TimeUnit.SECONDS.sleep(15);
```

Comment:
Can we make sure 15s is enough or shall we create a for loop query with sleep and max loop time until the initialization is completed?

Reply:
The `MLSyncUpCron` job run every 10s, increase to 15s  should good enough to wait for this job to complete and make master key initialized.

Reply:
```
[2024-01-29T18:44:43,764][INFO ][o.o.n.Node               ] [88665a43e3fa] started
[2024-01-29T18:44:43,807][INFO ][o.o.g.GatewayService     ] [88665a43e3fa] recovered [0] indices into cluster_state
[2024-01-29T18:44:53,752][INFO ][o.o.m.a.MLModelAutoReDeployer] [88665a43e3fa] Index not found, not performing auto reloading!
[2024-01-29T18:44:53,753][INFO ][o.o.m.c.MLCommonsClusterManagerEventListener] [88665a43e3fa] Starting ML sync up job...
[2024-01-29T18:45:03,786][INFO ][o.o.p.PluginsService     ] [88665a43e3fa] PluginService:onIndexModule index:[.plugins-ml-config/sZK8J9DtQC6TiFxNFcfmDA]
[2024-01-29T18:45:03,879][INFO ][o.o.c.m.MetadataCreateIndexService] [88665a43e3fa] [.plugins-ml-config] creating index, cause [api], templates [], shards [1]/[1]
```

Reply:
hmm, let me changed to retry as @zane-neo suggested